source/_integrations/fritz.markdown

```diff
@@ -60,16 +60,16 @@ Set a new password for the guest wifi.
 The password must be between 8 and 63 characters long.
 If no password is given, it will be auto-generated.
 
-| Data attribute | Optional | Description                                                                                                    |
-| ---------------------- | -------- | -------------------------------------------------------------------------------------------------------------- |
-| `device_id`            | no       | Only act on a specific  router                                                                                 |
-| `password`             | yes      | New password for the guest wifi                                                                                |
-| `length`               | yes      | Length of the auto-generated password. (_default 12_)                        |
+| Data attribute | Optional | Description                                           |
+| -------------- | -------- | ----------------------------------------------------- |
+| `device_id`    | no       | Only act on a specific  router                        |
+| `password`     | yes      | New password for the guest wifi                       |
+| `length`       | yes      | Length of the auto-generated password. (_default 12_) |
 
 ## Integration options
 
 It is possible to change some behaviors through the integration options.
-To change the settings, go to {% my integrations title="**Settings** > **Devices & services**" %}. Select the **AVM FRITZ!Box Tools** integration, then select **Configure**.
+To change the settings, go to {% my integrations title="**Settings** > **Devices & services**" %}. Select the **AVM FRITZ!Box Tools** integration, then select **Add**.
```

Comment:
I think this is also related to an already existing config entry, those related to the options flow?

Reply:
I'm glad you're more alert than me. Time to call it a day for me, it seems.